src/System.Net.Security/src/System/Net/Security/SecureChannel.cs

```diff
@@ -908,7 +909,8 @@ internal void ProcessHandshakeSuccess()
         --*/
         internal SecurityStatusPal Encrypt(ReadOnlyMemory<byte> buffer, ref byte[] output, out int resultSize)
         {
-            if (NetEventSource.IsEnabled)
+            bool logEnabled = NetEventSource.IsEnabled;
+            if (logEnabled)
```

Comment:
Why are we changing all the 'NetEventSource.IsEnabled' to 'logEnabled'? Is it really that much faster. 

We already a coding pattern everywhere else in networking code like this:

```c#
if (NetEventSource.IsEnabled) NetEventSource.method
```

Are we going to change it everywhere else in the codebase?

This change diminishes readability and the benefit in potential performance is questionable.

Reply:
Happy to revert; only adds 1 extra mov to the tests.